web/app/js/components/TapQueryForm.jsx

```diff
@@ -316,20 +325,20 @@ class TapQueryForm extends React.Component {
       <Grid container>
 
         <Grid container spacing={24}>
-          <Grid item xs={6} md={3}>
+          <Grid item xs={6} md={3} classes={{ item: classes.formControlWrapper }}>
```

Comment:
Yes, I can change it here because I only want to update the `root` element. Thanks.

@scottcarol, using `classes` doesn't mean to add multiple classes. Sometimes you need to access deeper elements and `className` is not enough, so you can use `classes` object prop to customise all the CSS injected. In fact, you do this here:

```HTML
<TableSortLabel
  active={active}
  direction={active ? order : col.defaultSortOrder || 'asc'}
  classes={{icon: active ? classes.activeSortIcon : classes.inactiveSortIcon}}
  onClick={this.createSortHandler(col)}>
  {col.title}
</TableSortLabel>

<Badge
  classes={{ badge: classes.badge }}
  invisible={this.state.hideUpdateBadge}
  badgeContent="1">
  <FontAwesomeIcon icon={faSmile} className={classes.shrinkIcon} />
</Badge>
```

Reply:
Thanks for making the change, and the explanation about `classes`. 🙏